airflow/docker-compose.yaml

```diff
@@ -79,6 +79,15 @@ x-airflow-common:
     # see https://googleapis.dev/python/google-auth/latest/user-guide.html#using-external-identities
     GOOGLE_CLOUD_PROJECT: cal-itp-data-infra
 
+    CALITP_BUCKET__AIRTABLE: "gs://test-calitp-airtable"
+#    CALITP_BUCKET__GTFS_RT_RAW: "gs://test-calitp-gtfs-rt-raw"
+#    CALITP_BUCKET__GTFS_RT_PARSED: "gs://test-calitp-gtfs-rt-parsed"
+#    CALITP_BUCKET__GTFS_RT_VALIDATION: "gs://test-calitp-gtfs-rt-validation"
+    CALITP_BUCKET__GTFS_SCHEDULE_RAW: "gs://test-calitp-gtfs-schedule-raw"
+    CALITP_BUCKET__GTFS_SCHEDULE_VALIDATION: "gs://test-calitp-gtfs-schedule-validation"
+
+    GRAAS_SERVER_URL: $GRAAS_SERVER_URL
```

Comment:
I think with #1690 we may not need this any more? maybe we can add a TODO, don't want to scope creep here too much without understanding ramifications but I literally just merged that